integration/containerd/cri/integration-tests.sh

```diff
@@ -122,25 +124,45 @@ create_containerd_config() {
 	if [ "${runtime_type}" == "${default_runtime_type}" ];then
 		local containerd_runtime=$(command -v "${runtime}")
 	fi
+	
+	if [ "${runtime_type}" == "${kata_runtime_type}" ];then
+		local containerd_runtime=${kata_runtime_bin}
+	fi
```

Comment:
Are these changes necessary up to here? AFAIU the `SHIMV2_TEST` variable already handles all of this.

Reply:
Ok, I have already delete this part.